examples/scripts/build-ios-on-mac

```diff
@@ -61,7 +61,7 @@ function build {
         pushd "${build_dir}"
         safe cmake \
             -DBUILD_SHARED_LIBS=ON \
-            -DCMAKE_TOOLCHAIN_FILE="${SCRIPT_DIR}/../cmake/external/ios-cmake/ios.toolchain.cmake" \
+            -DCMAKE_TOOLCHAIN_FILE="${SCRIPT_DIR}/../../cmake/external/ios-cmake/ios.toolchain.cmake" \
```

Comment:
How about to try to use CMake's toolchain? Separate commit?

Reply:
There used to be a ticket for this in another related repository.